modules/ROOT/pages/using-opscenter.adoc

```diff
@@ -56,7 +56,22 @@ sudo gravity planet enter -- --notty /usr/bin/gravity -- user create --ops-url=h
 After logging into Ops Center, you'll see a list of tabs on the left of the page. Navigate to "Monitoring"
 to view the dashboards available for viewing.
 
-=== View Metrics from an Application
+The dashboards are divided into the following categories:
+
+* *Applications* shows metrics on all the replicas for an application deployment. 
+** Filters include the environment ID the application is deployed in, and the application deployment name.
```

Comment:
```suggestion
** Filters include the environment ID the application is deployed in and the application deployment name.
```

Reply:
Suggestion: Filters include the ID for the environment where the application is deployed and the application deployment name.